packages/components/tests/dummy/app/templates/components/app-footer.hbs

```diff
@@ -0,0 +1,349 @@
+    <Shw::Text::H2>Content</Shw::Text::H2>
+
+    <button type="button" {{on "click" this.toggleHighlight}}>
+      {{if this.showHighlight "Hide" "Show"}}
+      layout highlight
+    </button>
+
+    <Shw::Text::H3>With only generic content</Shw::Text::H3>
+
+    <Hds::AppFooter as |AF|>
+      <AF.ExtraContentBefore>
+        <Shw::Placeholder @height="2em" @text="Extra Content Before" @background="#e4e4e4" />
+      </AF.ExtraContentBefore>
+      <AF.Item>
+        <Shw::Placeholder @height="2em" @text="Item" @background="#e4e4e4" />
+      </AF.Item>
+      <AF.Item>
+        <Shw::Placeholder @height="2em" @text="Item" @background="#e4e4e4" />
+      </AF.Item>
+      <AF.Item>
+        <Shw::Placeholder @height="2em" @text="Item" @background="#e4e4e4" />
+      </AF.Item>
+      <AF.ExtraContentAfter>
+        <Shw::Placeholder @height="2em" @text="Extra Content After" @background="#e4e4e4" />
+      </AF.ExtraContentAfter>
+    </Hds::AppFooter>
+
+    <Shw::Text::H3>With only the default content</Shw::Text::H3>
+
+    <Hds::AppFooter />
+
+    <Shw::Text::H3>With minimal recommended content</Shw::Text::H3>
+
+    <Hds::AppFooter as |AF|>
+      <AF.LegalLinks />
+    </Hds::AppFooter>
+
+    <Shw::Text::H3>With StatusLink & custom Link content</Shw::Text::H3>
+
+    <Hds::AppFooter as |AF|>
+      <AF.StatusLink @status="operational" />
+      <AF.Link @href="https://cloud.hashicorp.com/docs/changelog">Changelog</AF.Link>
+      <AF.LegalLinks />
+    </Hds::AppFooter>
+
+    <Shw::Text::H3>With custom “meta” items (non-link) content</Shw::Text::H3>
+
+    <Hds::AppFooter as |AF|>
+      <AF.StatusLink @status="operational" />
+      <AF.LegalLinks />
+      <AF.Item>
+        <Hds::Text::Body @tag="span" @size="100" @color="primary">v.01.15</Hds::Text::Body>
+      </AF.Item>
+      <AF.Item>
+        <Hds::Text::Body @tag="span" @size="100" @color="primary">Product Name</Hds::Text::Body>
+      </AF.Item>
+      <AF.Item>
+        <Hds::Text::Body @tag="span" @size="100" @color="primary">API: 1.02</Hds::Text::Body>
+      </AF.Item>
+    </Hds::AppFooter>
+
+    <Shw::Text::H3>With theme selector as Extra Content Before</Shw::Text::H3>
+
+    <Hds::AppFooter as |AF|>
+      <AF.ExtraContentBefore>
+        <Hds::Dropdown as |dd|>
+          <dd.ToggleButton @text="Theme" @color="secondary" @size="small" />
+          <dd.Interactive @icon="monitor" {{on "click" dd.close}} @text="System" />
+          <dd.Interactive @icon="moon" {{on "click" dd.close}} @text="Dark" />
+          <dd.Interactive @icon="sun" {{on "click" dd.close}} @text="Light" />
+        </Hds::Dropdown>
+      </AF.ExtraContentBefore>
+      <AF.StatusLink @status="operational" />
+      <AF.Link @href="https://cloud.hashicorp.com/docs/changelog">Changelog</AF.Link>
+      <AF.LegalLinks />
+    </Hds::AppFooter>
+
+    <Shw::Divider />
+
+    <Shw::Text::H2>Layout</Shw::Text::H2>
```

Comment:
Consider following the same approach/naming/labeling as in the `PageHeader` page, for consistency: https://hds-showcase.vercel.app/components/page-header (see "Component wrapping")

Reply:
It doesn't use container queries or break points though. The example widths follow the Figma examples but the exact pixel widths don't seem so vital for the AppFooter.

Reply:
This part has since been refactored to use `let` so resolving this as outdated.